myapp/templates/single_stock.html

```diff
@@ -33,124 +41,99 @@
-			<a class="nav-link" data-toggle="pill" href="#1y" role="tab" aria-controls="pills-contact" aria-selected="false">1Y</a>
-		  </li>
-			<li class="nav-item">
-			<a class="nav-link" data-toggle="pill" href="#max" role="tab" aria-controls="pills-contact" aria-selected="false">Max</a>
-		  </li>
-		</ul>
-		-->
-	</div>
+    <div class="pt-4 col-sm-6 pl-5">
+        <dl class="row text-left">
+            <dt class="col-sm-6">Previous Close</dt>
+            <dd class="col-sm-6">{{ data.previousClose }}</dd>
+
+            <dt class="col-sm-6">Volume</dt>
+            <dd class="col-sm-6">{{ data.volume }}
+            </dd>
+
+            <dt class="col-sm-6">52 Weeks Low</dt>
+            <dd class="col-sm-6">{{ data.week52Low }}</dd>
+
+            <dt class="col-sm-6">52 Weeks High</dt>
+            <dd class="col-sm-6">{{ data.week52High }}</dd>
+
+            <dt class="col-sm-6">Year Change</dt>
+            <dd class="col-sm-6">{{ data.ytdChange }}</dd>
+
+            <dt class="col-sm-6">Market Cap</dt>
+            <dd class="col-sm-6">{{ data.marketCap }}</dd>
+
+        </dl>
+    </div>
+    <div class="pt-4 col-sm-6">
+
+        <canvas id="stockChart"></canvas>
+                <form class="row offset-1" id="compareForm">
+            <input list="myDatalist" id="ex1" type="text"
+                   style="max-width:40%;max-height:10%;" name="stockNameToCompare">
+            <datalist id="myDatalist"
+                      style="max-height:10px;overflow-y:auto;display:block !important;">
+
+            </datalist>
+            <button type="submit" class="btn btn-primary">
+                <i class="material-icons" style="font-size:20px;">compare_arrows</i>
+                Compare
+            </button>
+        </form>
+        <!-- WIP TAB DESIGN
+        <ul class="nav nav-pills mb-3" id="pills-tab" role="tablist">
+          <li class="nav-item">
+            <a class="nav-link" data-toggle="pill" href="#1w" role="tab" aria-controls="pills-home" aria-selected="true">1W</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link active" data-toggle="pill" href="#1m" role="tab" aria-controls="pills-profile" aria-selected="false">1M</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" data-toggle="pill" href="#3m" role="tab" aria-controls="pills-contact" aria-selected="false">3M</a>
+          </li>
+            <li class="nav-item">
+            <a class="nav-link" data-toggle="pill" href="#6m" role="tab" aria-controls="pills-contact" aria-selected="false">6M</a>
+          </li>
+            <li class="nav-item">
+            <a class="nav-link" data-toggle="pill" href="#1y" role="tab" aria-controls="pills-contact" aria-selected="false">1Y</a>
+          </li>
+            <li class="nav-item">
+            <a class="nav-link" data-toggle="pill" href="#max" role="tab" aria-controls="pills-contact" aria-selected="false">Max</a>
+          </li>
+        </ul>
```

Comment:
A lot of work is needed here to ensure there are no merge conflicts between this and PR #32

Reply:
yeah, I will do this after you merge your PR.